packages/rebber/src/pre-visitors/abbr.js

```diff
@@ -0,0 +1,21 @@
+const all = require('../all')
+module.exports = plugin
+const identifiers = {}
+function plugin (ctx) {
+  return {
+    abbrVisitor: abbrVisitor,
+    txtVisitor: txtVisitor
+  }
+  function abbrVisitor (node, index, parent) {
+    if (node.referenceType === 'shortcut') { // remark for abbr
+      identifiers[node.identifier] = all(ctx || {}, node)
```

Comment:
It'd be better to have `all(ctx, node` here and have a default parameter in the function definition: `function plugin (ctx = {}) {`